src/main/groovy/com/blackducksoftware/integration/hub/detect/bomtool/DockerBomTool.groovy

```diff
@@ -63,22 +71,30 @@ class DockerBomTool extends BomTool {
 
     @Override
     public List<DependencyNode> extractDependencyNodes() {
-        URL hubDockerInspectorShellScriptUrl = new URL("https://blackducksoftware.github.io/hub-docker-inspector/hub-docker-${detectProperties.dockerInspectorVersion}.sh")
```

Comment:
The location of the hub-docker-inspector script should be configurable, either to  a local file or a URL of their choosing. This will help work around network issues